src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2215,6 +2217,7 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public override int GetHashCode() { throw null; }
         public int GetHashCode(System.StringComparison comparisonType) { throw null; }
         public int IndexOf(char value) { throw null; }
+        public int IndexOf(char value, StringComparison comparisonType) { throw null; }
```

Comment:
Are there tests for this new `IndexOf` overload?

Reply:
yah

Reply:
I missed them. Can you point me to them?

Reply:
src/System.Runtime/tests/System/StringTests.netcoreapp.cs

Reply:
I’m sorry I can’t seem to find them. I must be missing something. Can you provide a direct link?  I see the new Contains tests you added, but no tests for the new IndexOf overload.

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System/StringTests.netcoreapp.cs#L141

Reply:
@Anipik, that's a test of Contains, not IndexOf.  @eerhardt is asking about the new IndexOf overload, for which no new tests were added.

Reply:
Sorry you are right. I am not directly testing indexof. I will add tests

Reply:
@stephentoub  should they be added  to stringtests.cs or stringtests.netcoreapp.cs?

Reply:
> should they be added to stringtests.cs or stringtests.netcoreapp.cs?

They won't compile if they're added to StringTests.cs.  They should be added to StringTests.netcoreapp.cs.